### DISCUSSION TITLE
Please add multiline recognition for the quotes

### DISCUSSION THREAD

**Discussion by Sashkow** [jordwest/news-feed-eradicator#374] [Ideas]
I wanted to paste poems, but the lines were scrambled into one continuous line. Could you render the quotes preserving newline breaks?

**Comment by theshakilahmed** [comment]
This would make the extension more versatile, although it will change the current formatting rules.
One interesting thing would be allowing to display widgets. example widget: BTC price, to-do list, date time

**Comment by Sashkow** [comment]
How about something like this: if no '\n' then old formatting rules, else new formatting rules.
